proto/sample_v2_round_trip.json

```diff
@@ -933,6 +933,25 @@
             "scoreDisplayMode": "not_applicable", 
             "title": "`<input type=\"image\">` elements have `[alt]` text"
         }, 
+        "installable-manifest": {
```

Comment:
interesting why did only these two swap order?

Reply:
looks like they're alphabetized. No idea why, but I guess it doesn't matter :) @exterkamp